src/ert/async_utils.py

```diff
@@ -74,6 +76,6 @@ def _done_callback(task: asyncio.Task[_T_co]) -> None:
         if (exc := task.exception()) is None:
             return
 
-        logger.error(f"Exception occurred during {task.get_name()}", exc_info=exc)
```

Comment:
The reason this logging message could never be seen originally is because `ert.async_utils` is missing in `logger.conf`